com.kms.cura.dal/resources/SQL Script/Appointment.sql

```diff
@@ -27,3 +27,6 @@ INSERT INTO Appointments (doctor_id, patient_id, facility_id, appt_day, start_ti
 INSERT INTO Appointments (doctor_id, patient_id, facility_id, appt_day, start_time, end_time, current_status) VALUES (3, 2, 8, 20160830, '08:00:00', '09:00:00', 6);
 INSERT INTO Appointments (doctor_id, patient_id, facility_id, appt_day, start_time, end_time, current_status) VALUES (3, 2, 8, 20160830, '15:00:00', '16:30:00', 6);
 INSERT INTO Appointments (doctor_id, patient_id, facility_id, appt_day, start_time, end_time, current_status) VALUES (3, 2, 8, 20160830, '13:00:00', '14:30:00', 6);
+
+alter table appointments add rate int;
+alter table appointments add rate_cmt text;
```

Comment:
This is the seed database file. Why not write these 2 columns straight into the create table step ?